### DISCUSSION TITLE
ExpressRoute

### DISCUSSION THREAD

**Discussion by Piotr2016** [AzureCommunityPL/group#439] [Pomoc]
Witajcie.
Szykuje się do egzaminu AZ-900 i ćwicząc pytania natknołem się na 
![image](https://user-images.githubusercontent.com/23290616/174969978-7dc2c2b1-50ec-4b43-bcb3-c339941bb9f9.png)

Z moich doświadczeń wynika, że ExpressRoute to usługa na łączu dzierżawionym, a takowe obecnie zakończone są interfejsem Eth, więc odpowiedź C że jest to Site-to-Site VPN jest błędna. Co jako starzy wyjadacze o tym sądzicie ?

**Comment by smajchrz** [comment]
To egzamin Microsoft wiec pytania są o ich produkty. Site to Site VPN jest jednym z nich.

Swoją drogą materiały na Microsoft Learn są wystarczające aby zdać ten egzamin.

**Comment by TomaszOledzki** [comment]
> Z moich doświadczeń wynika, że ExpressRoute to usługa na łączu dzierżawionym, a takowe obecnie zakończone są interfejsem Eth, więc odpowiedź C że jest to Site-to-Site VPN jest błędna. Co jako starzy wyjadacze o tym sądzicie ?

ExpressRoute możesz mieć na dedyku albo na łączu współdzielonym. Ruch nie jest szyfrowany - więc puszczanie ruchu w tunelu IPsec np. przez VPN S2S ma sens w przypadku ExpressRoute.

Edit:
OnPrem - ISP - Microsoft Edge (Azure)

ExpressRoute zestawisz między OnPrem - ISP. Natomiast ruch ISP - Microsoft Edge leci współdzielonym łączem przez klientów Twojego ISP (z vlan tag). Pytanie jak bardzo ufasz swojemu ISP i od kogo od poddzierżawia kable.

Może w planie ExpressRoute DirectConnect jest inaczej - tutaj chyba masz dedykowane łącze OnPrem - ISP - Microsoft Edge (Azure).
